pkgs/by-name/go/goatcounter/package.nix

```
@@ -0,0 +1,54 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+, testers
+, goatcounter
+}:
+
+buildGoModule rec {
+  pname = "goatcounter";
+  version = "2.5.0";
+
+  src = fetchFromGitHub {
+    owner = "arp242";
+    repo = pname;
```

```suggestion
    repo = "goatcounter";
```

please in general avoid using `pname`, as it can be overridden